src/sentry/utils/outcomes.py

```diff
@@ -74,9 +74,8 @@ def track_outcome(
 
     outcomes_config = kafka_config.get_topic_definition(settings.KAFKA_OUTCOMES)
     billing_config = kafka_config.get_topic_definition(settings.KAFKA_OUTCOMES_BILLING)
-    use_billing = (
-        outcome.is_billing() and settings.KAFKA_TOPICS[settings.KAFKA_OUTCOMES_BILLING] is not None
-    )
+
+    use_billing = outcome.is_billing() and outcomes_config == billing_config
```

Comment:
This whole function seems to be flipping between topics and clusters for some reason I haven't quite grasped yet. It could probably use a refactor, but for now I'm keeping the previous behavior.